data-tck/src/main/java/io/micronaut/data/tck/entities/Genre.java

```diff
@@ -0,0 +1,19 @@
+package io.micronaut.data.tck.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Genre extends GenericEntity<Long> {
```

Comment:
You can just leave it as it was without the generic entity, even so, Sonar complains

Reply:
Updated